LibreNMS/ComposerHelper.php

```diff
@@ -44,7 +44,10 @@ public static function postInstall(Event $event)
             self::setPermissions();
         }
 
+        // "optimize" doesn't clear view cache etc.
+        self::exec('php artisan optimize:clear');
```

Comment:
maybe we should do a `view:clear` only here, since `optimize` does some cleaning itself.